cli/src/command/account.rs

```diff
@@ -677,7 +682,8 @@ pub async fn send_native_token_command(
                 TokenId::from_str(&token_id)?,
                 U256::from_dec_str(&amount).map_err(|e| Error::Miscellaneous(e.to_string()))?,
             )],
-            ..Default::default()
+            return_address: Default::default(),
+            expiration: Default::default(),
```

Comment:
Just curious, personal preference or technical issue?

Reply:
There is no default address and native tokens, so the previous default didn't make sense there